src/Helpers.elm

```diff
@@ -53,9 +53,50 @@ decimals digits num =
     digits
         |> toFloat
         |> (^) 10
-        |> (*) num
+        |> (*) (abs num)
         |> round
         |> splitThousands
         |> String.concat
         |> String.right digits
         |> String.padLeft digits '0'
+
+
+{-| Returns the sign of a converted number
+
+    >>> sign 1 "1" "0"
+    ""
+
+    >>> sign 0 "0" "0"
+    ""
+
+    >>> sign -1 "1" "0"
+    "-"
+
+    >>> sign 0 "0" "000"
+    ""
+
+    >>> sign -0.01 "0" "0"
+    ""
+
+    >>> sign -0.01 "0" "01"
+    "-"
+-}
+sign : Float -> String -> String -> String
+sign num integers decimals =
```

Comment:
What about a `type alias` to make this signature more meaningful? Something among the lines of:

```elm
type alias FormattedNumber =
    { original : Float
    , integers: String
    , decimals: String
    , prefix: Maybe String  -- or we can call it just `signal`
    }

This looks good and can be helpful in other functions too. What do you think about it?